utils/gnupg/Makefile

```diff
@@ -37,6 +38,12 @@ define Package/gnupg
   MENU:=1
 endef
 
+define Package/gpgv
+  $(call Package/gnupg/Default)
+  TITLE+= (signature verification only)
+  MENU:=1
```

Comment:
Ok, fixed.  I've also shortened the gpgv package title so it's more likely visible in menuconfig than the old one.